sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/RelationshipSamples.cs

```diff
@@ -21,36 +21,57 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
             // Despite not being a good code practice, this prevents code snippets from being out of context for the user when making API calls that accept Ids as parameters.
 
             PrintHeader("RELATIONSHIP SAMPLE");
+            string sampleBuildingModelId = "dtmi:com:samples:SampleBuilding;1";
+            string sampleFloorModelId = "dtmi:com:samples:SampleFloor;1";
+            try
+            {
+                Console.WriteLine($"Deleting model Id '{sampleBuildingModelId}' if it exists.");
+                await client.DeleteModelAsync(sampleBuildingModelId);
+            }
+            catch (RequestFailedException ex) when (ex.Status == 404)
+            {
+                // Model did not exist yet, and that's fine
+            }
+
+            try
```

Comment:
Do you think this would read better if this try/catch block was a helper method?

It seems to me it isn't central to the reading of the sample, but more about smooth operation of a sample.